app/graphql/types/invoices/status_type_enum.rb

```diff
@@ -5,7 +5,7 @@ module Invoices
     class StatusTypeEnum < Types::BaseEnum
       graphql_name 'InvoiceStatusTypeEnum'
 
-      Invoice::STATUS.reject { |k| k == :generating }.each do |type|
+      Invoice::VISIBLE_STATUS.keys.each do |type|
```

Comment:
😍

Reply:
Yeah, I think it's really what we're trying to communicate when we exclude `generating`